.buildkite/publish/publish-common.sh

```diff
@@ -18,11 +18,34 @@ source $SCRIPT_DIR/git-setup.sh
 VERSION_PATH="$PROJECT_ROOT/connectors/VERSION"
 export VERSION=$(cat $VERSION_PATH)
 
+# Variables for build.yaml
+version="${VERSION}"
+version_qualifier="${VERSION_QUALIFIER:-}"
+revision="$(git rev-parse HEAD)"
+repository="$(git config --get remote.origin.url)"
+
 if [[ "${USE_SNAPSHOT:-}" == "true" ]]; then
+  # SNAPSHOT workflows ignore version qualifier
   echo "Adding SNAPSHOT labeling"
   export VERSION="${VERSION}-SNAPSHOT"
+  version="${VERSION}"
```

Comment:
oh this is nice, now we'll get `*-SNAPSHOT` output for snapshot artifacts too! :feelsgood: